packages/server/src/model/database.ts

```
@@ -17,11 +17,7 @@ export const runSql = async <T extends mysql.RowDataPacket>(
 
     return rows
   } catch (error) {
-    const errorMessage = `Error executing SQL query: ${
-      (error as Error).message
-    }`
-    log.error(errorMessage)
-    throw new Error(errorMessage)
+    captureErrorEvent({ error: error as Error })
```

I would return a Error here or an error type